File: Lecture05/ListViews/app/src/main/java/com/codingblocks/listviews/MainActivity.java
package com.codingblocks.listviews;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.ListView;

import com.codingblocks.listviews.adapters.CourseAdapter;
import com.codingblocks.listviews.models.Course;

import java.util.ArrayList;

public class MainActivity extends AppCompatActivity {

    ArrayList<Course> courses = Course.genRandomCourses(50);

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        ListView lvCourses = findViewById(R.id.lvCourses);
        CourseAdapter courseAdapter = new CourseAdapter(this, courses);
        lvCourses.setAdapter(courseAdapter);

    }
}

File: Lecture06/KotlinSample/app/src/main/java/com/codingblocks/kotlinsample/MainActivity.kt
package com.codingblocks.kotlinsample

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.support.v7.widget.LinearLayoutManager
import android.support.v7.widget.RecyclerView
import com.codingblocks.kotlinsample.adapters.CourseAdapter
import com.codingblocks.kotlinsample.models.Course
import com.codingblocks.kotlinsample.models.genCourses
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    val courses: ArrayList<Course> = genCourses(100)

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        rvCourses.layoutManager = LinearLayoutManager(this)
        rvCourses.adapter = CourseAdapter(this, courses)

    }
}
